amoro-ams/amoro-ams-server/pom.xml

```diff
@@ -283,6 +290,22 @@
             <artifactId>flink-runtime-web</artifactId>
         </dependency>
 
+        <!--  apache hudi dependencies -->
+        <dependency>
```

Comment:
Adding this template into dependencyManagement may simplify the configuration of sub modules.

Reply:
This dependency has different configuration in ams-server and ams-core